test/expected/outputWithPartial.html

```diff
@@ -1,3 +1,4 @@
 <h2>gulp-ejs</h2>
 
-<h3><p>Hello rpvl!</p></h3>
+<h3><p>Hello rpvl!</p>
+</h3>
```

Comment:
Because this is the actual rendering of `withpartial.ejs` by the current version of ejs which this module depends now.

Please see this commit (https://github.com/rogeriopvl/gulp-ejs/commit/8ff732f734087fca4e4fd95bb7eaf70bdd57d4ea). At that point, ejs rendering was changed and the workaround (of removing line breaks) was introduced, but I don't see any reason to keep that workaround. So I removed the workaround and fixed the fixture like the above.

I hope this makes sense.